core/src/OpenMP/Kokkos_OpenMP_Instance.cpp

```diff
@@ -256,8 +256,6 @@ void OpenMPInternal::initialize(int thread_count) {
       }
     }
 
-    OpenMP::memory_space space;
```

Comment:
What's the deal with that change?

Reply:
It's not used which was revealed by defaulting the Host space constructor.

Reply:
It is not clear to me how defaulting the default constructor changes things but this is fine.